ui/lib/core/addon/components/confirmation-modal.hbs

```diff
@@ -3,40 +3,45 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<Modal @title={{@title}} @onClose={{@onClose}} @isActive={{@isActive}} @type={{this.type}} @showCloseButton={{true}}>
-  <section class="modal-card-body">
-    {{yield}}
-    <div class="modal-confirm-section">
-      <p class="has-text-weight-semibold is-size-6">
-        Confirm
-      </p>
-      <p class="sub-text has-top-bottom-margin-xxs">Type
-        <strong>{{this.confirmText}}</strong>
-        to confirm
-        {{this.toConfirmMsg}}</p>
-      <Input
-        @type="text"
-        @value={{this.confirmationInput}}
-        name="confirmationInput"
-        class="input has-margin-top"
-        autocomplete="off"
-        spellcheck="false"
-        data-test-confirmation-modal-input={{or @title true}}
-      />
-    </div>
-  </section>
-  <footer class="modal-card-foot modal-card-foot-outlined">
-    <button
-      type="button"
-      class="button {{this.buttonClass}}"
-      disabled={{not-eq this.confirmationInput this.confirmText}}
-      {{on "click" @onConfirm}}
-      data-test-confirm-button={{or @title true}}
-    >
-      {{this.buttonText}}
-    </button>
-    <button type="button" class="button is-secondary" {{on "click" @onClose}} data-test-cancel-button>
-      Cancel
-    </button>
-  </footer>
-</Modal>
\ No newline at end of file
+{{#if @isActive}}
```

Comment:
<img width="729" alt="Screenshot 2023-10-03 at 10 53 17 AM" src="https://github.com/hashicorp/vault/assets/68122737/3d731656-8b50-402b-a542-a5d763358c3e">
<img width="749" alt="Screenshot 2023-10-03 at 10 53 27 AM" src="https://github.com/hashicorp/vault/assets/68122737/c44d6163-451a-4418-9881-7411b5448556">

Reply:
Can this modal appear in both instances you have in the screenshots (with or without the header) or is the second screen what it looked like before?

Reply:
Gotcha thanks!